lib/src/mesh_network.dart

```diff
@@ -9,6 +9,7 @@ abstract class IMeshNetwork {
   Future<String> get name;
   Future<List<ProvisionedMeshNode>> get nodes;
   String get id;
+  Future<List<dynamic>> get provisionersUUIDList;
```

Comment:
Would be better to have a type instead of dynamic

Reply:
if i do List<String> then i've got type 'List<dynamic>' is not a subtype of type 'List<String>' error.

Reply:
why use `List<dynamic>` if the type is `List<String>` ?

Reply:
i was using List String and then i got this error. but i will look into it again